pythran/pythonic/core.hpp

```diff
@@ -1,6 +1,11 @@
 #ifndef PYTHONIC_CORE_HPP
 #define PYTHONIC_CORE_HPP
 
+#define QUOTEME(M) #M
```

Comment:
there's already a similar macro in sources/pythran/pythran/pythonic/include/types/numpy_nary_expr.hpp

```
#define STR_(a) #a
```